newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -977,6 +1008,30 @@ export const LeaderboardAdmin = ({ onLoading, project }: Props) => {
               }}
             />
           ) : null}
+          {isEditingSortOptions ? (
+            <LeaderboardSortOptionsDialog
+              open
+              onClose={() => setIsEditingSortOptions(false)}
+              onSave={async (sortOptions: {|
+                sort: LeaderboardSortOption,
+                extremeAllowedScore: ?number,
+              |}) => {
+                try {
+                  await onUpdateLeaderboard(i18n, {
+                    ...sortOptions,
+                  });
+                } finally {
+                  setIsEditingSortOptions(false);
+                }
+              }}
+              sort={currentLeaderboard ? currentLeaderboard.sort : undefined}
```

Comment:
You can put this check with `{isEditingSortOptions ?` because it is non-sense otherwise